app/services/planning_application_search.rb

```diff
@@ -83,7 +83,7 @@ def my_applications
   end
 
   def all_applications
-    @all_applications ||= local_authority.planning_applications.includes([:application_type]).by_created_at_desc
+    @all_applications ||= local_authority.planning_applications.includes([:application_type])
```

Comment:
Are you expecting the planning applications to be sorted by application type? The `includes` will most likely do a second query to load the application type and the applications will be unsorted

Reply:
Funny enough, I am getting Bullet Warnings to delete the `includes` and when I am deleting it I am getting another warning to add it back